plugins/woocommerce-admin/client/layout/index.js

```diff
@@ -185,8 +185,15 @@ function _Layout( {
 		);
 	}
 
-	const { breadcrumbs, layout = { header: true, footer: true } } = page;
-	const { header: showHeader = true, footer: showFooter = true } = layout;
+	const {
+		breadcrumbs,
+		layout = { header: true, footer: true, transientNotices: true },
+	} = page;
+	const {
+		header: showHeader = true,
+		footer: showFooter = true,
+		transientNotices: showTransientNotices = true,
```

Comment:
I'm not so sold on introducing an API that allows transient notices to be removed on certain pages.  I know that this is used to manually add it back into the footer on product pages, but this could lead to losing this feature on certain pages if we're not careful.

If we want notices to respect fixed footer areas and this is something we may run into in the future, should we create a footer slot and fill both the notices and and fixed bars inside this area for all screens?

Reply:
> this could lead to losing this feature on certain pages if we're not careful.

True, but it is opt-out.

> If we want notices to respect fixed footer areas and this is something we may run into in the future, should we create a footer slot and fill both the notices and and fixed bars inside this area for all screens?

I'll explore this approach tomorrow in an alternate PR tomorrow.